src/app/fdctl/config/default.toml

```diff
@@ -580,6 +580,13 @@ dynamic_port_range = "8900-9000"
         # so this should be unused.
         tx_buf_size = 4096
 
+        # QUIC has a concept of an "idle" connection - one where neither the
+        # client nor the server has sent any packet to the other.
+        # An idle connection will be terminated if it remains idle longer than
+        # some threshold. "idle_timeout_millis" represents this threshold in
+        # milliseconds
+        idle_timeout_millis = 2500
```

Comment:
I think logan added this so a rebase remove this diff. But can you keep your comment? it's a little nicer